CircleCI

```diff
@@ -0,0 +1 @@
+
```

Comment:
// Echo.java
class Echo
{
   public static void main (String [] args) throws java.io.IOException
   {
      int ch;
      System.out.print ("Enter some text: ");
      while ((ch = System.in.read ()) != '\n')
         System.out.print ((char) ch);
   }
}